packages/ui-tree-browser/src/TreeBrowser/TreeButton/index.tsx

```diff
@@ -93,8 +93,8 @@ class TreeButton extends Component<Props> {
     selected: PropTypes.bool,
     focused: PropTypes.bool,
     level: PropTypes.number,
-    containerRef: function () {},
-    renderContent: function () {}
```

Comment:
This seemed to be buggy, build:types failed on it. Changed it to `PropTypes.func`, because that is what we use everywhere else.